src/components/TopBar/Wallet/hooks/useFaucet.ts

```diff
@@ -0,0 +1,53 @@
+import { t } from "@lingui/macro";
+import { ContractReceipt } from "ethers";
+import { useMutation } from "react-query";
+import { useDispatch } from "react-redux";
+import { DEV_FAUCET } from "src/constants/addresses";
+import { useDynamicFaucetContract } from "src/hooks/useContract";
+import { error as createErrorToast, info as createInfoToast } from "src/slices/MessagesSlice";
+
+export const useFaucet = () => {
+  const dispatch = useDispatch();
+  const contract = useDynamicFaucetContract(DEV_FAUCET, true);
+
+  return useMutation<ContractReceipt, Error, string>(
+    async token_ => {
+      if (!contract)
+        throw new Error(
+          t`Give is not supported on this network. Please switch to a supported network, such as Ethereum mainnet`,
```

Comment:
"Faucet", not "Give". Also change the suggested network.

Reply:
✅